packages/frontend/src/components/RichTextEditor/MenuBar.tsx

```diff
@@ -19,23 +20,56 @@ import {
   RiTableLine,
   RiUnderline,
 } from 'react-icons/ri'
+import { LoadingButton } from '@mui/lab'
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material'
```

Comment:
is it possible to use chakra for this? cus we wanna move out of mui right

Reply:
Hmm we don't have a modal component using chakra yet leh, I think let's solve it in another PR